pom.xml

```diff
@@ -223,7 +204,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-aether-provider</artifactId>
-      <version>${mavenVersion}</version>
+      <version>3.3.9</version>
```

Comment:
The released version of Maven already depends on Maven Core 3.5.0, but Maven Plugin bundles core 3.1.0. So we are already in the YOLO mode :(

maven-aether-provider 3.5.0 does not exist. If the new library uses different package names, there should be no issue. We could bundle both. What could possibly go wrong? (c)